docker-compose.yml

```diff
@@ -10,8 +10,6 @@ services:
       - "5433:5432"
     volumes:
       - ./db/worker.sql:/docker-entrypoint-initdb.d/schema.sql
-    depends_on:
```

Comment:
Only call explicitly made to the documentation container is done in the `make package`